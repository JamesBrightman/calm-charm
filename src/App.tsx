import React from "react";
import { BrowserRouter, Routes, Route } from "react-router-dom";
import { HomePage } from "./HomePage";
import { Test } from "./Test";
import { Authentication } from "./Authentication";

export const App = () => {
  return (
    <BrowserRouter>
      <Routes>
        <Route path="auth/*" element={<Authentication />} />
        <Route path="/" element={<HomePage />} />
        <Route path="test" element={<Test />} />
      </Routes>
    </BrowserRouter>
  );
};
