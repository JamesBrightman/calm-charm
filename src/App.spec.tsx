import React from 'react';
import { render } from '@testing-library/react';
import { App } from './App';

describe('App component', () => {
  it('Component renders', () => {
    const { baseElement } = render(<App />);

    expect(baseElement).toBeTruthy();
  });

  it('Component matches snapshot', () => {
    const { baseElement } = render(<App />);

    expect(baseElement).toMatchSnapshot();
  });
});
