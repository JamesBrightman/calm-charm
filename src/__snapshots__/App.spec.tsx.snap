// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component Component matches snapshot 1`] = `
<body>
  <div>
    <div>
      Loading
    </div>
  </div>
</body>
`;
